app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
+        payload[:content_items].map.with_index do |event, index|
+          create_content_item(event, index, payload[:content_id])
```

Comment:
This could get us in a bit of a mess I think

As it creates a new user_facing_version each time this would only really work if you sent over a sequence of states that were "superseded" (x times), "publish". If you had "draft", "published" it'd get in a mess - I'd expect you'd get a validation message, but that'd be quite hard to decipher as it won't relate to which item in your import caused the problem.